### PR TITLE
Feature/team

### PR DESCRIPTION
**Pull request by glissario**
animation of memberpictures + 2 "white fields" where the team is hiding and camp + fire is build

### CODE REVIEW

File: src/components/FullTeam/FullTeam.vue [hunk 1]
```diff
@@ -1,7 +1,30 @@
...
+      @mouseleave="showTent = false"
+    >
+      <transition name="fadetent">
+        <img v-show="showTent" class="camptent" src="@/assets/tent.webp" />
+      </transition>
+    </div>
+
+    <div class="team" id="team">
+      <div v-for="(member, id) in shuffle(members)" v-bind:key="id">
+        <TeamMember v-show="showFullTeam" class="member" :member="member" />
```
Comment (robinboehm) [inline src/components/FullTeam/FullTeam.vue:18]: Team page re-renders and re-orders every time the fire is shown. Can we make the easter-egg a bit less destructive? :D

**Review by robinboehm — CHANGES_REQUESTED** [review]
Small changes. Then ready to merge. I like the texts =)

File: src/components/FullTeam/FullTeam.vue (hunk 1 above)
Comment (glissario) [reply src/components/FullTeam/FullTeam.vue:18]: Are there any other requests than the destructive animation? :D I have deleted them for now and will discuss the case in the group - tbh its not my wanted solution

**Comment by robinboehm** [comment]
Ready to merge. Do it @glissario =)

**Pull request merged**